src/links/airlines/aaSabre.js

```diff
@@ -0,0 +1,227 @@
+import mptUserSettings, { registerSetting } from "../../settings/userSettings";
+import { printNotification, iataToUtcOffset } from "../../utils";
+import { validatePaxcount, registerLink } from "../../print/links";
+import { currentItin } from "../../parse/itin";
+import apTimeZones from "../../json/timezones.json";
+
+const aaSabreEditions = [
+  { value: "BS", name: "Bahamas (USD)" },
+  { value: "VG", name: "British Virgin Islands (USD)" },
+  { value: "CA", name: "Canada (CAD)" },
+  { value: "PR", name: "Puerto Rico (USD)" },
+  { value: "GB", name: "United Kingdom (GBP)" },
+  { value: "US", name: "United States (USD)" }
+];
+
+function printAaSabre() {
+  var dateToEpoch = function(y, m, d) {
```

Comment:
Could we also remove this code now? I don't think its used anymore

Reply:
Yes, I intended to go back and remove it but forgot... good catch. I've deleted it to keep everything nice and clean.